### ISSUE TITLE
Flutter 2.10 & 3.0 throw warnings during compilation

### ISSUE THREAD

**Issue by ciriousjoker** [RafaelBarbosatec/tutorial_coach_mark#118]
Example:

```dart
WidgetsBinding.instance!.addObserver(this);
```

now becomes (notice the missing `!`):

```dart
WidgetsBinding.instance.addObserver(this);
```

**Comment by phcs971** [comment]
Fixed in #119, waiting for merge, you can use
`dependency_overrides:
  tutorial_coach_mark:
    git:
      url: https://github.com/phcs971/tutorial_coach_mark
      ref: master`
in the meanwhile

**Comment by RafaelBarbosatec** [comment]
Problem fixed in 1.2.3 version !
